.buildkite/scripts/steps/functional/e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- Trigger kibana e2e pipeline"
+ts-node .buildkite/scripts/steps/trigger_pipeline.ts kibana-e2e-tests "$BUILDKITE_BRANCH" "$BUILDKITE_COMMIT"
```

Comment:
Where is this step being used if what seems to be called is `.buildkite/pipelines/e2e.yml` at `.buildkite/pipelines/on_merge.yml`?

Reply:
Pushed [41bf6c0](https://github.com/elastic/kibana/pull/152846/commits/41bf6c0eef1f8ab82264375acefb5035fd430a5a)